[
  {
    "url": "https://api.github.com/repos/ipython/ipython/issues/comments/30593214",
    "html_url": "https://github.com/ipython/ipython/issues/4695#issuecomment-30593214",
    "issue_url": "https://api.github.com/repos/ipython/ipython/issues/4695",
    "id": 30593214,
    "node_id": "MDEyOklzc3VlQ29tbWVudDMwNTkzMjE0",
    "user": {
      "login": "takluyver",
      "id": 327925,
      "node_id": "MDQ6VXNlcjMyNzkyNQ==",
      "avatar_url": "https://avatars3.githubusercontent.com/u/327925?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/takluyver",
      "html_url": "https://github.com/takluyver",
      "followers_url": "https://api.github.com/users/takluyver/followers",
      "following_url": "https://api.github.com/users/takluyver/following{/other_user}",
      "gists_url": "https://api.github.com/users/takluyver/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/takluyver/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/takluyver/subscriptions",
      "organizations_url": "https://api.github.com/users/takluyver/orgs",
      "repos_url": "https://api.github.com/users/takluyver/repos",
      "events_url": "https://api.github.com/users/takluyver/events{/privacy}",
      "received_events_url": "https://api.github.com/users/takluyver/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2013-12-14T22:46:52Z",
    "updated_at": "2013-12-14T22:46:52Z",
    "author_association": "MEMBER",
    "body": "Yes, I expect that would break other things.\n\nI guess you've enabled saving output history? That's off by default, so it doesn't get so much exercise.\n"
  },
  {
    "url": "https://api.github.com/repos/ipython/ipython/issues/comments/30593302",
    "html_url": "https://github.com/ipython/ipython/issues/4695#issuecomment-30593302",
    "issue_url": "https://api.github.com/repos/ipython/ipython/issues/4695",
    "id": 30593302,
    "node_id": "MDEyOklzc3VlQ29tbWVudDMwNTkzMzAy",
    "user": {
      "login": "cpcloud",
      "id": 417981,
      "node_id": "MDQ6VXNlcjQxNzk4MQ==",
      "avatar_url": "https://avatars1.githubusercontent.com/u/417981?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/cpcloud",
      "html_url": "https://github.com/cpcloud",
      "followers_url": "https://api.github.com/users/cpcloud/followers",
      "following_url": "https://api.github.com/users/cpcloud/following{/other_user}",
      "gists_url": "https://api.github.com/users/cpcloud/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/cpcloud/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/cpcloud/subscriptions",
      "organizations_url": "https://api.github.com/users/cpcloud/orgs",
      "repos_url": "https://api.github.com/users/cpcloud/repos",
      "events_url": "https://api.github.com/users/cpcloud/events{/privacy}",
      "received_events_url": "https://api.github.com/users/cpcloud/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2013-12-14T22:51:12Z",
    "updated_at": "2013-12-14T22:51:28Z",
    "author_association": "NONE",
    "body": "Slightly OT: A fresh clone of ipython throws ~10 errors re autoreloading the `email` module during a call to `nosetests`. Is that a bug?\n"
  },
  {
    "url": "https://api.github.com/repos/ipython/ipython/issues/comments/30596256",
    "html_url": "https://github.com/ipython/ipython/issues/4695#issuecomment-30596256",
    "issue_url": "https://api.github.com/repos/ipython/ipython/issues/4695",
    "id": 30596256,
    "node_id": "MDEyOklzc3VlQ29tbWVudDMwNTk2MjU2",
    "user": {
      "login": "takluyver",
      "id": 327925,
      "node_id": "MDQ6VXNlcjMyNzkyNQ==",
      "avatar_url": "https://avatars3.githubusercontent.com/u/327925?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/takluyver",
      "html_url": "https://github.com/takluyver",
      "followers_url": "https://api.github.com/users/takluyver/followers",
      "following_url": "https://api.github.com/users/takluyver/following{/other_user}",
      "gists_url": "https://api.github.com/users/takluyver/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/takluyver/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/takluyver/subscriptions",
      "organizations_url": "https://api.github.com/users/takluyver/orgs",
      "repos_url": "https://api.github.com/users/takluyver/repos",
      "events_url": "https://api.github.com/users/takluyver/events{/privacy}",
      "received_events_url": "https://api.github.com/users/takluyver/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2013-12-15T01:55:27Z",
    "updated_at": "2013-12-15T01:55:27Z",
    "author_association": "MEMBER",
    "body": "To run the tests, you'll need to install IPython and run `iptest` (or call `IPython.test()`). There are some complexities that just running nosetests doesn't take account of, including running the autoreload tests in a separate process.\n"
  }
]
